### ISSUE TITLE
Multiple / default-all plugins in subcommands

### ISSUE THREAD

**Issue by stuartpb** [plushu/plushu-plugins-plugin#6]
## plugins:install

Would need a separate plugin to take multiple, as currently plugins:install takes the plugin URL, followed by an optional name to use when installing.

Alternately, could be changed to a format that takes multiple plugins, and takes specified names with a `--as` flag (I think this might be how NPM does it). Still, `plugins:install` taking one plugin name isn't a major disaster, as it's easily put in a `for` loop, and it's the same signature as the Heroku toolbelt (and `git clone`).

(described in plushu/plushu#17)
## plugins:uninstall

Could take multiple. If failure / `--force` get implemented, this should maybe be re-examined.
## plugins:update

Could take multiple. Defaulting to all (a la the Heroku toolbelt) _could_ be allowed, but I feel like that could be asking for trouble. (On the other hand, `packer -Syu` makes lots of changes in one sweep, and it gets by.)

Basically, before updating _anything_ you should check the Google group to see what's incompatible with what, etc.
## plugins:disable

Could take multiple.

**Comment by stuartpb** [comment]
It should still really be implemented for `install`, with `--as`.

Indeed, just plugins:install should have its own discussion.
